pyproject.toml

```
@@ -40,7 +40,7 @@ autoflake = "^2.2.1"
 isort = "^5.13.2"
 markdownify = "^0.11.6"
 tavily-python = "^0.3.9"
-microchain-python = "^0.4.4"
+microchain-python = { git = "https://github.com/galatolofederico/microchain.git", rev = "56db91de72b6466080c26434631808cc20af670c" }
```

Using latest microchain revision until [this feature](https://github.com/galatolofederico/microchain/pull/17) makes its way into the release

---

```suggestion
microchain-python = { git = "https://github.com/galatolofederico/microchain.git", rev = "98e601f6b7413ea48fb0b099309d686c4b10ff5c" }
```

This is the latest now 🙏